RELEASING.md

```diff
@@ -0,0 +1,59 @@
+## Versioning guidelines
+
+Dagster versions follow the guidelines in [PEP 440](https://www.python.org/dev/peps/pep-0440//).
+
+To make dependency management easier in the context of a monorepo with many installable projects,
+package versions move in lockstep with each other and with git tags. 
+
+As the API is still in flux, we aren't following strict semver rules at this point, but roughly
```

Comment:
might just say "semantic versioning" instead of "semver". I had to stop, think and then reverse map to that. I had never seen semver